MAINTAINING.md

```diff
@@ -0,0 +1,91 @@
+# Maintenance releases
+
+## Merging the Pull Request & releasing a new version
+
+Releases are automated using [semantic-release](https://github.com/semantic-release/semantic-release).
+The following commit message conventions determine which version is released:
+
+1. `fix: ...` or `fix(scope name): ...` prefix in subject: bumps fix version, e.g. `1.2.3` → `1.2.4`
+2. `feat: ...` or `feat(scope name): ...` prefix in subject: bumps feature version, e.g. `1.2.3` → `1.3.0`
+3. `BREAKING CHANGE:` in body: bumps breaking version, e.g. `1.2.3` → `2.0.0`
+
+Only one version number is bumped at a time, the highest version change trumps the others.
+Besides, publishing a new version to npm, semantic-release also creates a git tag and release
+on GitHub, generates changelogs from the commit messages and puts them into the release notes.
+
+Before the publish it runs the `npm run build` script which creates a `pkg/` folder with distributions for browsers, node and Typescript definitions. The contents of the `pkg/` folder are published to the npm registry.
+
+If the pull request looks good but does not follow the commit conventions, use the <kbd>Squash & merge</kbd> button.
+
+> ⚠️ making sure the message is semantic-release compliant before clicking <kbd>Confirm squash and merge</kbd>:
+
+![Screenshot of GitHub's Squash and Merge confirm dialog](assets/squash-and-merge-dialog.png)]
+
+## Maintenance releases
+
+### 0. Requirements
+
+`semantic-release` is configured in the `package.json` of each repository. If `release.branches` is set, make sure that it includes the line for maintenance releases, for example
+
+```js
+  "release": {
+    "branches": [
+      "+([0-9]).x",
+      "main",
+      "beta"
+    ],
+```
+
+`semantic-release` is run in the`.github/workflows/release.yml` GitHub Action workflow. Make sure it's triggered on push in the `*.x` release branches.
+
+```yml
+name: Release
+"on":
+  push:
+    branches:
+      - main
+      - next
+      - beta
+      - "*.x"
+```
+
+### 1. Create a branch for the maintenance version
+
+Find the latest version that was released on the maintenance version. For example, if the current version is 3.1, and you want to release maintenance versions for 2.x, then find the latest 2.x version. Say that's 2.10.9. In that example, create a tag based on this tag
```

Comment:
Change `In that example, create a tag based on this tag` to `In that example, create a branch based on this tag`